modules/experimental/cilium-block-egress.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "cilium.io/v2"
```

Comment:
Maybe this file should go into `catapult/kube/airgap/` to be cleaner, all yaml files are there.

Reply:
I wanted to keep experimental work in the experimental folder. Even though I think this is fairly stable now, I think more people would need to use it to be promoted to a stable api feature